internal/mediainfo/dvd.go

```
@@ -139,38 +171,89 @@ func parseDVDVideo(path string, file *os.File, size int64, opts AnalyzeOptions)
+	payloadFileSize := int64(0)
 	if aggregateMode {
 		if vobPaths, vobSize := dvdTitleSetVOBs(path); len(vobPaths) > 0 && vobSize > 0 {
-			info.FileSize = vobSize
+			if len(backupData) == 0 && len(vobPaths) > 1 {
+				vobPaths = vobPaths[:1]
+				if firstInfo, err := os.Stat(vobPaths[0]); err == nil { //nolint:gosec // path came from root-bounded DVD grammar discovery
+					vobSize = firstInfo.Size()
+				}
+			}
+			aggregateSize := vobSize
 			if ifoInfo, err := os.Stat(path); err == nil {
```

golangci-lint (2.12.2)

[high] G703: Path traversal via taint analysis

(gosec)